src/qt/editaddressdialog.cpp

```diff
@@ -2,6 +2,8 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
+#include <wallet/wallet.h>
```

Comment:
Could declare g_address_type as extern if you wanted to remove this.

Reply:
Done, but wasn't able to do this in receivecoinsdialog (`undeclared identifier 'OUTPUT_TYPE_BECH32`, etc).